stablehlo/reference/Ops.cpp

```diff
@@ -2065,8 +2084,8 @@ Tensor reduceScatterOp(const Tensor &operand, int64_t scatterDimension,
       allReduceOp(operand, replicaGroups, channelId, useGlobalDeviceIds, region,
                   process, scope, operand.getType());
 
-  auto parts = split(reducedValue, processGroups[0].size(), scatterDimension,
-                     operand.getType().getContext());
+  auto parts = split(reducedValue.front().getTensor(), processGroups[0].size(),
```

Comment:
This is safe to call `front` because we only will ever call this with a single operand, I assume?

Reply:
we need getTensor(). front() will return `InterpreterValue`,  and split expects `Tensor`.